Eduria/Eduria/Views/Exam/Create.cshtml

```diff
@@ -48,12 +49,13 @@
                 var link = question.MediaLink ?? "none";
                 var type = (MediaType)question.MediaType;
                 var questionType = (QuestionType)question.QuestionType;
-                var typeInt = question.MediaType;
+                var typeInt = (int)question.MediaType;
```

Comment:
Werkt dit nog gewoon? Aangezien we een MediaSourceId gebruiken nu?

Reply:
Als het goed is wel. Ondanks dat Eduriadata/models/Question gebruik maakt van een MediaSourceId worden de type en source uit de mediasource opgeslagen in de Eduria/models/QuestionModel. De laatste model wordt gebruikt in Create.cshtml